content/management/configuration-api/v3.3/index.mdx

```diff
@@ -3621,50 +3690,52 @@ Registers a webhook for the Client ID (application) provided in the request. Lat
-| [`routing_status_set`](#routing_status_set)                          |                                                 - |
-| [`agent_deleted`](#agent_deleted)                                    |                    `chat_member_ids`, `agent_ids` |
-| [`incoming_customer`](#incoming_customer)                            |                                                 - |
-| [`customer_session_fields_updated`](#customer_session_fields_updated)|                                 `chat_member_ids` |
-| [`events_marked_as_seen`](#events_marked_as_seen)                    |                `chat_member_ids`, `only_my_chats` |
+| [`incoming_chat`](#incoming_chat)                                    |                `chat_presence`, `only_my_chats` |
+| [`chat_deactivated`](#chat_deactivated)                              |                `chat_presence`, `only_my_chats` |
+| [`chat_access_granted`](#chat_access_granted)                        |                `chat_presence`, `only_my_chats` |
+| [`chat_access_revoked`](#chat_access_revoked)                        |                `chat_presence`, `only_my_chats` |
+| [`user_added_to_chat`](#user_added_to_chat)                          |                `chat_presence`, `only_my_chats` |
+| [`user_removed_from_chat`](#user_removed_from_chat)                  |                `chat_presence`, `only_my_chats` |
+| [`incoming_event`](#incoming_event)                                  | `chat_presence`, `author_type`, `only_my_chats` |
+| [`event_updated`](#event_updated)                                    | `chat_presence`, `author_type`, `only_my_chats` |
+| [`incoming_rich_message_postback`](#incoming_rich_message_postback)  |                `chat_presence`, `only_my_chats` |
+| [`chat_properties_updated`](#chat_properties_updated)                |                `chat_presence`, `only_my_chats` |
+| [`chat_properties_deleted`](#chat_properties_deleted)                |                `chat_presence`, `only_my_chats` |
+| [`thread_properties_updated`](#thread_properties_updated)            |                `chat_presence`, `only_my_chats` |
+| [`thread_properties_deleted`](#thread_properties_deleted)            |                `chat_presence`, `only_my_chats` |
+| [`event_properties_updated`](#event_properties_updated)              |                `chat_presence`, `only_my_chats` |
+| [`event_properties_deleted`](#event_properties_deleted)              |                `chat_presence`, `only_my_chats` |
+| [`thread_tagged`](#thread_tagged)                                    |                `chat_presence`, `only_my_chats` |
+| [`thread_untagged`](#thread_untagged)                                |                `chat_presence`, `only_my_chats` |
+| [`routing_status_set`](#routing_status_set)                          |                                               - |
+| [`agent_deleted`](#agent_deleted)                                    |                    `chat_presence`, `agent_ids` |
```

Comment:
@litwinow in the code this WH didn't have `chat_member_ids` at all (so no `chat_presence` either), bug in docs or code?

Reply:
ah, bug in docs, will fix